h2o-py/tests/testdir_munging/pyunit_6987_merge_one_empty.py

```diff
@@ -0,0 +1,22 @@
+from __future__ import print_function
+import sys
+sys.path.insert(1,"../../")
+import h2o
+from tests import pyunit_utils
+import random
+
+def mergeOneEmptyFrame():
+    # PUBDEV-6987: merge with one empty frame and one normal frame.
```

Comment:
if one of these frames is empty, do we need an empty csv in small data?

Reply:
The csv contains the column names in this case.  It is not empty.